telegram/ext/updater.py

```diff
@@ -52,8 +52,8 @@ class Updater:
 
     Note:
         * You must supply either a :attr:`bot` or a :attr:`token` argument.
-        * If you supply a :attr:`bot`, you will need to pass :attr:`defaults` to *both* the bot and
-          the :class:`telegram.ext.Updater`.
+        * If you supply a :attr:`bot`, you will need to pass :attr:`arbitrary_callback_data`,
+          and :attr:`defaults` to the bot instead of the :class:`telegram.ext.Updater`.
```

Comment:
I added an "… to the bot instead of the :class:`telegram.ext.Updater`. In this  case, you'll have to use the class :class:`telegram.ext.ExtBot`." to this note

Docs & type hints don't need to be updated IMO because `ExtBot` is a subclass of `Bot` …